pkg/imgpkg/bundle/fake_images_metadata_builder_test.go

```diff
@@ -127,41 +86,37 @@ func (r *FakeImagesMetadataBuilder) WithImageFromPath(imageNameFromTest string,
 		r.t.Fatalf("Failed trying to build a file image%s", err)
 	}
 
-	r.updateState(imageNameFromTest, fileImage, path)
-	reference, err := name.ParseReference(imageNameFromTest)
-	if err != nil {
-		r.t.Fatalf("Failed trying to get image name: %s", err)
-	}
-
-	return r.state[reference.Name()]
+	return r.updateState(imageNameFromTest, fileImage, path)
 }
 
-func (r *FakeImagesMetadataBuilder) CleanUp() {
+func (r *FakeTestRegistryBuilder) CleanUp() {
 	for _, tarPath := range r.state {
 		os.Remove(filepath.Join(tarPath.path, ".imgpkg", "images.yml"))
 	}
+	if r.server != nil {
+		r.server.Close()
+	}
+}
+
+func (r *FakeTestRegistryBuilder) relocateToTestServer(repo string) string {
```

Comment:
```suggestion
func (r *FakeTestRegistryBuilder) referenceOnTestServer(repo string) string {
```

Since there are no relocations in `imgpkg` maybe it makes sense to call relocation to this method. The above is a suggestion.

Reply:
Looking at the rest of the functions, is there any reason for this function to be private?

Reply:
Good catch. Thx I made the change